.github/workflows/ci.yaml

```diff
@@ -1,8 +1,12 @@
 on:
   push:
-    branches: [main]
+    paths-ignore:
```

Comment:
I'm not sure if `paths-ignore` is necessary, there aren't many commits where you only change `.md` or `.gitignore` and we only save some cents for microsoft 🙄 So I would rather have the CI config simpler

Reply:
If you do not want the CI to run twice for a PR when it is an internal PR then i think the only way would be to do an if condition check to make sure the two repository sources are different so in my case it would be a difference of luigi311 and lit-robotics and im not sure if that would be worth the cluttering.